sdk/metrics/src/main/java/io/opentelemetry/sdk/metrics/exemplar/ExemplarReservoir.java

```diff
@@ -0,0 +1,59 @@
+/*
+ * Copyright The OpenTelemetry Authors
+ * SPDX-License-Identifier: Apache-2.0
+ */
+
+package io.opentelemetry.sdk.metrics.exemplar;
+
+import io.opentelemetry.api.common.Attributes;
+import io.opentelemetry.context.Context;
+import io.opentelemetry.sdk.metrics.data.Exemplar;
+import java.util.Collections;
+import java.util.List;
+
+/**
+ * An interface for an exemplar reservoir of samples.
+ *
+ * <p>This represents a reservoir for a specific "point" of metric data.
+ */
+public interface ExemplarReservoir {
+
+  /** Offers a measurement to be sampled. */
+  public void offerMeasurementLong(long value, Attributes attributes, Context context);
```

Comment:
Remove `public` from methods since interface

Reply:
It's probably ok to call them both `offerMeasurement` for better readability

Reply:
Done.